drivers/arm/i2c_master.c

```diff
@@ -102,7 +102,7 @@ uint8_t i2c_readReg(uint8_t devaddr, uint8_t* regaddr, uint8_t* data, uint16_t l
 }
 
 // This is usually not needed. It releases the driver to allow pins to become GPIO again.
-uint8_t i2c_stop(uint16_t timeout)
+uint8_t i2c_stop(void)
```

Comment:
@jackhumbert or @skullydazed, is this okay?

Reply:
It looks like it doesn't actually use this timeout parameter, at all. 

So it looks like this should be safe.

Reply:
I just noticed the comment on the function is garbage though, i2c_stop is part of the wire protocol, it's not "releasing the driver". I'll delete that when I get a chance today.